docs: translate the CONTRIBUTING in english (#62)
ineat/refcards: CONTRIBUTING.md

# Contribution

## Repository organization

> The repository contains RefCards concerning technologies. There must be one directory per Technology. If needs are different by technology, "sub-directories" must be created first.

Reviewer: ```suggestion
The repository contains refcards concerning some technologies. One directory must exist by technology. If needs are different by technology, sub-directories must be created first.
```

Author: Ok

The `README.md` lists all the technologies available on the repository. Any new technology must be added there with the link to each of the language files.

> ## Naming convention

Reviewer: ```suggestion
## Naming conventions
```

Naming and typography conventions:
***Directories*
* Root directories are written in lowercase

> * The directories must contain the versions of the RefCards by language 

Reviewer: ```suggestion
* The directories must contain the refcard versions, by language 
```

**Files**

> * The files are named by language `ISO 639-1` en extension `.md`

Reviewer: ```suggestion
* The files are named by language, according to the `ISO 639-1` spec and the extension must be `.md`
```

**README.md**
* The `README.md` must include each technology present
* Technologies must display available languages

> * Languages must point to the files in question

Reviewer: ```suggestion
* Languages must point to the related files
```

└── etc...
```

## Changes Publications 
The contribution must be allong the new branch some repository. 

> The branch make change will be reviewed and merged by another contibutor referent on the technology.

Reviewer: ```suggestion
The branch make change will be reviewed and merged by an another maintainer of the technology.
```

> ## Revues

Reviewer: ```suggestion
## Reviews
```

> Don’t forgot that the Refcards they ar part of technology showcase of company. The review it avoided misspelling.

Reviewer: ```suggestion
Don’t forgot that the refcards are parts of company technology showcase. The review allows to avoid some misspellings for example.
```
